### PR TITLE
Small update to OpenMaxHelpCommand

### PR DESCRIPTION
**Pull request by ross-g**
This should open the help Url specific to the Max version being used, falling back to 2018 if it cannot be found.

### CODE REVIEW

**Comment by cb109** [comment]
Hi Ross, this somehow slipped under my radar, sorry! Will take a look at it some time this week, thank you for your contribution.

File: MAXAutoIntel.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2014, Luca Faggion
```
Comment (cb109) [inline MAXAutoIntel.py:1]: Whoops, I think I forgot to add a commit to `master` that removes this file (I refactored it to `sublime3dsmax.Completions` a while ago. Will fix that.

File: sublime3dsmax.py [hunk 2]
```diff
@@ -135,6 +136,26 @@ def _send_cmd_to_max(cmd):
...
 
 
+def _get_max_version():
+    """ Try to determine the version of 3ds Max that we are interacting with.
+    """
+    global mainwindow
+    # default to 2018 help, this has the most updated docs and will filter to Maxscript results
+    max_version = "2018"
+
+    if mainwindow is None:
```
Comment (cb109) [inline sublime3dsmax.py:146]: I'd prefer to just always get the window anew, no matter if we already fetched it, makes for simpler code and we can always be sure to have it up to date.

EDIT: Actually forget about that, I forgot that we may have connected to a specific Max instance and that could break.

File: sublime3dsmax.py [hunk 3]
```diff
@@ -249,7 +270,10 @@ def run(self, edit):
             if not word.empty():
                 key = self.view.substr(word)
                 query_param = "?query=" + key
-                url = constants.ONLINE_MAXSCRIPT_HELP_URL + query_param
+                max_version = _get_max_version()
+                url = constants.ONLINE_MAXSCRIPT_HELP_URL[max_version] + query_param
+                if max_version == "2018":
+                    url += r"&cg=Scripting%20%26%20Customization"
```
Comment (cb109) [inline sublime3dsmax.py:276]: I think it would be nice to have a dedicated `get_help_url()` method that also explains in its docstring what this does.

Comment (cb109) [inline sublime3dsmax.py:276]: Thank you for your contribution Ross! I'll do the small changes myself quickly, based on your commits :+1:

**Comment by ross-g** [comment]
Hey @cb109 

Wondering if I could request that you merge your develop branch to master and push a release? I configured a new workstation recently with my usual Sublime setup and had to replicate your changes from this PR manually, as the Package Control version of Sublime3dsMax was never updated.

Thanks!

**Comment by cb109** [comment]
Hey @ross-g 

Yea I never released that for some reason :see_no_evil: Pushed as 0.10.0, package control should pick it up soon. Please let me know if the installation worked, it's always good to have some confirmation.

**Comment by ross-g** [comment]
Yep that's worked thanks, Package Control updated this morning and I get the correct help for my Max version. There was one (non serious) syntax warning:
`sublime3dsmax.py:142: SyntaxWarning: name 'mainwindow' is used prior to global declaration`

**Comment by cb109** [comment]
Cool, thanks! Will fix that warning for the next release.

**Pull request merged**